### PR TITLE
update read me

### PR DESCRIPTION
**Pull request by Ashu002**
Please merge it

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,3 @@
 # test-repo
+
+This repo for learning purpose
```
Comment (sp7592) [inline README.md:3]: please correct the repo meaning

Comment (Ashu002) [reply README.md:3]: repo is short name so it is correct please merge it

Comment (sp7592) [inline README.md:3]: plz correct it

Comment (Ashu002) [reply README.md:3]: ok Done

**Pull request merged**